test/IRGen/variadic_generics.sil

```diff
@@ -74,35 +74,34 @@ bb0:
   return %index : $Builtin.PackIndex
 }
 
-// CHECK-LABEL: define{{.*}} @test_pack_element_get_1(
```

Comment:
I don't think you meant to delete my test case here. :)

Reply:
Oops!